client/lib/abtest/active-tests.js

```diff
@@ -4,6 +4,11 @@
  * paths here (e.g. `import * from '../../lib/`)
  */
 
+/**************************************************************************************************/
+/* This library is deprecated! Please consider ExPlat for your next A/B experiment.               */
+/* See pbxNRc-sZ-p2 or visit #a8c-experiments and ask around!
```

Comment:
It'd also be good to change these links to point to the new component readme, given that it's open source and seen by people outside a8c (not that they'd be running A/B tests, but still...).